tools/testsys/src/aws_resources.rs

```diff
@@ -81,15 +108,23 @@ impl AwsK8s {
                 .await?,
             )
             .to_string();
-        let eks = self.eks_crd(testsys_images)?;
-        let ec2 = self.ec2_crd(testsys_images, Some(ami))?;
+        let eks = self.eks_crd(cluster_name, testsys_images)?;
+        let ec2 = self.ec2_crd(cluster_name, testsys_images, Some(ami))?;
+        let instance_provider = ec2.name().unwrap();
```

Comment:
In a previous forced push you replaced `unwrap` with `expect`, but you missed two, I left a comment in the other.